fip-6.md

```diff
@@ -0,0 +1,141 @@
+---
+fip: 6
+title: Provide ability to deactivate FIO Addresses and Domains
+status: Draft
+type: Functionality
+author: Casey Gardiner <casey@dapix.io>
+created: 2020-04-22
+updated: 
+---
+
+## Abstract
+This FIP implements the following:
+* Adds the ability for FIO Address owners the ability to deactivate stale or unused addresses.
+* Adds the ability for FIO Domain owners the ability to set their domain as expired.
+* Adds new API end points for FIO Domain deactivation and FIO Address burn. 
+* Adds new fees for FIO Domain deactivation and FIO Address burn.
```

Comment:
So is it burn or deactivate. I feel like deactivate implies it will be around and can be reactivated.

Reply:
I agree.  I will make the changes to better describe what is actually happeneing

Reply:
Technically, if they renew their domain or address during the 90 grace period, they are essentially reactivating it, yes?